backend/src/backup/backup_bulk.rs

```diff
@@ -330,15 +332,14 @@ async fn perform_backup<Db: DbHandle>(
                 &manifest.volumes,
             )
             .await;
-        drop(guard);
+        guard.unmount().await?;
```

Comment:
I assume this is because `async` drop traits are not a thing?

Reply:
Yes

Reply:
https://github.com/bluss/scopeguard
Would the above help?